__tests__/__fixtures__/one-module-one-async-beforeeach-new-import-input-test.js

```diff
@@ -0,0 +1,25 @@
+import {module, test} from 'qunit';
+import {setupApplicationTest} from 'ember-qunit';
+
+const SELECTORS = Object.freeze({
+  MOCK_SELECTOR: '[data-test-nav-bar-browse]',
+});
+
+module('Acceptance | browse acceptance test', function (hooks) {
```

Comment:
I'm not sure this matters. These are test fixtures, and never really intended to be read (and are also in the __fixtures__ dir).

Reply:
Thanks for the comment. Are you referring to the fixture's file name? If so, I will keep the file name as is for now, to be consistent with the rest of the test fixture names. Eventually I'd like to break apart these test fixtures if possible so that they are representing and testing only a specific function of the plugin transformation. I'd also like to make these file name references more dynamic.